lesson_12/homework.md

```diff
@@ -142,29 +143,30 @@ But, the provided code for these classes and their instances has several mistake
 related to hierarchy, class attributes, and instance creation.
 
 Find and correct these mistakes to make the code work properly.
+**Leave a comment in the code explaining what the problem was and why it wouldn't work.**
```

Comment:
In the py file we also indicated that there are 7 mistakes in the code. I would suggest that we aim for consistency and either keep or remove it everywhere. 
Since the code is quite long and topic is quite difficult, I think knowing the number of mistakes might be really helpful for the students. What do you think?

Reply:
Added clarifying note in MD file.